<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>820</width>
    <height>648</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="margin">
     <number>0</number>
    </property>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_3" stretch="4,1">
      <property name="leftMargin">
       <number>5</number>
      </property>
      <property name="rightMargin">
       <number>5</number>
      </property>
      <item>
       <widget class="PhVideoView" name="videoView" native="true"/>
      </item>
      <item>
       <widget class="PhGraphicStripView" name="stripView" native="true"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>820</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionOpen_Video"/>
   </widget>
   <widget class="QMenu" name="menuControl">
    <property name="title">
     <string>Control</string>
    </property>
    <widget class="QMenu" name="menuVarial_Speed">
     <property name="title">
      <string>Varial Speed</string>
     </property>
     <addaction name="action_3"/>
     <addaction name="action_1"/>
     <addaction name="action_0_5"/>
     <addaction name="action0"/>
     <addaction name="action0_5"/>
     <addaction name="action1"/>
     <addaction name="action3"/>
    </widget>
    <addaction name="actionPlay_pause"/>
    <addaction name="actionPlay_backward"/>
    <addaction name="actionStep_forward"/>
    <addaction name="actionStep_backward"/>
    <addaction name="actionStep_time_forward"/>
    <addaction name="actionStep_time_backward"/>
    <addaction name="menuVarial_Speed"/>
    <addaction name="actionGo_To"/>
    <addaction name="actionSet_Time_Code"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuControl"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen">
   <property name="text">
    <string>Open...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionPlay_pause">
   <property name="text">
    <string>Play/pause</string>
   </property>
   <property name="shortcut">
    <string>Up</string>
   </property>
  </action>
  <action name="actionPlay_backward">
   <property name="text">
    <string>Play backward</string>
   </property>
   <property name="shortcut">
    <string>Down</string>
   </property>
  </action>
  <action name="actionStep_forward">
   <property name="text">
    <string>Step forward</string>
   </property>
   <property name="shortcut">
    <string>Right</string>
   </property>
  </action>
  <action name="actionStep_backward">
   <property name="text">
    <string>Step backward</string>
   </property>
   <property name="shortcut">
    <string>Left</string>
   </property>
  </action>
  <action name="actionStep_time_forward">
   <property name="text">
    <string>Step time forward</string>
   </property>
   <property name="shortcut">
    <string>Alt+Right</string>
   </property>
  </action>
  <action name="actionStep_time_backward">
   <property name="text">
    <string>Step time backward</string>
   </property>
   <property name="shortcut">
    <string>Alt+Left</string>
   </property>
  </action>
  <action name="action_3">
   <property name="text">
    <string>-3</string>
   </property>
  </action>
  <action name="action_1">
   <property name="text">
    <string>-1</string>
   </property>
  </action>
  <action name="action_0_5">
   <property name="text">
    <string>-0.5</string>
   </property>
  </action>
  <action name="action0">
   <property name="text">
    <string>0</string>
   </property>
  </action>
  <action name="action0_5">
   <property name="text">
    <string>0.5</string>
   </property>
  </action>
  <action name="action1">
   <property name="text">
    <string>1</string>
   </property>
  </action>
  <action name="action3">
   <property name="text">
    <string>3</string>
   </property>
  </action>
  <action name="actionGo_To">
   <property name="text">
    <string>Go to...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+G</string>
   </property>
  </action>
  <action name="actionOpen_Video">
   <property name="text">
    <string>Open Video</string>
   </property>
  </action>
  <action name="actionSet_Time_Code">
   <property name="text">
    <string>Set Time Code</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PhGraphicStripView</class>
   <extends>QWidget</extends>
   <header>PhGraphicStrip/PhGraphicStripView.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>PhVideoView</class>
   <extends>QWidget</extends>
   <header>PhVideo/PhVideoView.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
